frontend/src/components/Card.tsx

```diff
@@ -98,11 +103,7 @@ const Card = ({
                 />
               ))}
           </div>
-          {projectName && (
-            <a href={projectLink} target="_blank" rel="noopener noreferrer">
-              {projectName}
-            </a>
-          )}
+
           {(languages || (topics && topics.length > 0) || (social && social.length > 0)) && (
```

Comment:
For issues card we also want to make sure we show topics. I believe they are already added here in the card itself, but I don't see any right now. Could you please check why is that? The topics should go under the languages labels. Screenshot of the same card at port 8000 and this PR:
![Screenshot 2024-12-23 at 8 27 56 AM](https://github.com/user-attachments/assets/3cab7d40-ed73-4691-92b8-17993c0d97eb)
![Screenshot 2024-12-23 at 8 28 00 AM](https://github.com/user-attachments/assets/3fa74070-395f-40f1-b594-62048357242a)